tests/scripts/mocksocktests.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
```

Comment:
This is the testcase? can we handle all cluster nodes in one testcasefile?

Reply:
Currently this is the test case, yes.

In this script, we can start multiple instances of `mocksock.pl` as background processes, each with its expected traffic. Then, `wait` (bash builtin) waits for all of them to finish. Another option is to have a simulated Redis cluster which can handle multiple ports and connections, implemented in C with libevent, but dumb and controlled expected traffic commands in a similar way as mocksock.pl.

The idea I have is that this script (or another script) can loop over a number of test files, which are not code but only the redis commands, expected output from the client and the expected traffic. Either we make it three files per test case (`test-case-42.commands`, `test-case-42.expected-output` and `test-case-42.expected-traffic`) or we combine them into one file on some way.